common/src/main/java/io/apicurio/registry/model/GA.java

```diff
@@ -0,0 +1,15 @@
+package io.apicurio.registry.model;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+@EqualsAndHashCode
+public class GA {
```

Comment:
nitpick: `GroupAndArtifact` or `GroupAndArtifactId`, `GA` often refers to "General Availability" and it's slightly misleading ... I guess ...

Reply:
Yeah understood.  We have new `GA` and `GAV` classes on `main` for Registry 3, so I used the same name to make porting this feature to 3.0 easier.  But we'll give this nitpick some thought.  :)